src/Microsoft.AspNetCore.Server.IntegrationTesting/project.json

```diff
@@ -46,12 +46,8 @@
         "Microsoft.Web.Administration": "7.0.0"
       },
       "frameworkAssemblies": {
-        "System.Net.Http": "",
-        "System.Runtime": {
-          "type": "build"
-        },
-        "System.Xml": "",
-        "System.Xml.Linq": ""
+        "System.Xml": "4.0.0.0",
+        "System.Xml.Linq": "4.0.0.0"
```

Comment:
Should we be concerned about the NETStandard.Library System.Xml.XDocument framework assembly reference to System.Xml.Linq?